src/styles/_school.scss

```diff
@@ -0,0 +1,3 @@
+.school-page {
+  margin-top: 80px;
```

Comment:
@amygroshek nit: `$spacer` is .8rem=8px. could we make this `10 * $spacer` -- or, if this should adjust based on `StickyHeader`, we could do `calc(#{$sticky-header-height} + (2 * #{$spacer}`? (I think you'll need `calc` since it's px + rem)

Reply:
Thanks for the info. Please just merge this so I can get working.